src/axom/sidre/examples/sidre_createdatastore.cpp

```diff
@@ -415,6 +415,60 @@ void generate_blueprint(DataStore* ds) {
   // _blueprint_generate_save_end
 }
 
+void generate_blueprint_to_path(DataStore* ds) {
```

Comment:
@nselliott -- could you please add a description to the top of the file that explains the different meshes that are generated/output by this example?

Reply:
I've added a brief description of every function called from main().